NebulaPatcher/Patches/Dynamic/PlanetTransport_Patch.cs

```diff
@@ -35,18 +36,36 @@ public static bool SetStationStorage_Postfix(PlanetTransport __instance, int sta
         // as the game does not do that for some reason, we need to do it here
         [HarmonyPostfix]
         [HarmonyPatch("NewStationComponent")]
-        public static void NewStationComponent_Postfix(PlanetTransport __instance, StationComponent __result, int _entityId, int _pcId, PrefabDesc _desc)
+        public static void NewStationComponent_AddPlanetId_Postfix(PlanetTransport __instance, StationComponent __result, int _entityId, int _pcId, PrefabDesc _desc)
```

Comment:
Personally I think it makes sense for each patch that does something seperate, seperate, so it's easier to see what is actually relevant to look at and give them a name for what they're doing.

Especially on a Postfix, its kind of two eventlisteners doing things instead of one large eventlistener.

On a prefix you could do the same and have a fixed order by using priority annotations too.

I don't mind merging them though.

Same with the name, It's basically, "BaseMethod_WhatItDoes_Type". Anyone else with input?

Reply:
I agree with @Jon-Myrstad. We gain maintainability by having separate patches for different functional components.

Reply:
@sp00ktober Also, I think it's actually in some cases helpful to have the PlanetID available so you always can do a lookup on StationId + PlanetId for both PLS and ILS's if you need to, I actually think some of the processors are dependant on this now.

It's really weird to me that its not set to anything by game code, I understand the devs don't use it, but why not set it?